sklearn/tests/test_naive_bayes.py

```diff
@@ -695,6 +695,19 @@ def test_categoricalnb():
         assert_array_equal(clf.predict(np.array([[0, 0]])), np.array([2]))
 
 
+def test_unseen_categories_at_predict():
+    X, y = make_classification(n_features=10, n_classes=3, n_samples=1000,
+                               random_state=42, n_redundant=0, n_informative=6)
+    X = np.abs(X.astype(np.int))
+    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.5,
```

Comment:
This would be clearer if you just explicitly created some toy data that should raise this error.

Reply:
Thanks for the review. Addressed in https://github.com/scikit-learn/scikit-learn/pull/16045/commits/a470a4adc4be64ccb5c999a4eba4f494e7459b7a